fgpv-vpgf/fgpv-vpgf
# feat(ui): enhance loading indicators

## Description
Enhanced the service loading feedback. See #2794 for details.

Closes #2794

## Testing
- open `config-01` in `index-samples`
- a notifications that "Liquid pipelines" (response time for this layer is set to 100) are slow should pop up - a similar notifications for the "WMS Layer" should be displayed next
- enable "crops" layer (response time is default 4000)
- a notification for "crops" will display
- zoom or pan the map
- notifications for "crops" is display every time the layer redraws

If you don't get notifications for "crops", [throttle your network in dev tools](https://developers.google.com/web/tools/chrome-devtools/network-performance/network-conditions).

## Documentation
Lots of inline comments

## Checklist



Remember, it is a *muffin offence* to open a PR with any of the above checklist items incomplete.

Please keep the original issue up to date with the final solution, expected behaviour, and any additional notes for testers


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/fgpv-vpgf/fgpv-vpgf/2799)
